.changeset/kind-news-tell.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/odsp-driver": minor
+---
+---
+"section": other
+---
+
+Removed dependency on node-fetch
+
+The dependency on `node-fetch` has been removed in favour of native-fetch, which is available in all major browsers. Since v18 of Node, it doesn't require any extra flags when invoking the `node` executable. This should reduce Fluid's bundle size.
```

Comment:
````suggestion
The `@fluidframework/odsp-driver` package had a dependency on [node-fetch](https://www.npmjs.com/package/node-fetch) to provide consistent behavior of the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) across Node.js and browsers. Since v18 of Node.js, the Node-native Fetch API implementation no longer requires extra flags to be enabled, so the Fetch API is effectively now natively available on all browsers and Node.js. 

This dependency removal should reduce Fluid Framework's contribution to application bundle sizes.
````